query-node/mappings/content-directory/entity/index.ts

```diff
@@ -290,21 +273,17 @@ async function contentDirectory_EntityPropertyValuesUpdated(db: DB, event: Subst
 
   const { 2: newPropertyValues } = extrinsic.args
   const entityId = decode.stringIfyEntityId(event)
-
-  const ce = await db.get(ClassEntity, { where: { id: entityId } })
-  if (ce === undefined) throw Error(`Class not found for the entity id: ${entityId}`)
-
-  const cls = contentDirectoryClassNamesWithId.find((c) => c.classId === ce.classId)
-  if (cls === undefined) throw Error(`Not known class id: ${ce.classId}`)
-
   const where: IWhereCond = { where: { id: entityId } }
 
+  const knownClass = await getKnownClass(db, where)
+  if (!knownClass) return
```

Comment:
Here again, if we got an `EntityPropertyValuesUpdated` event how come we don't have the entity in the database?
We are effectively ignoring the event and our database is inconsistent with the runtime?

Reply:
Had a call with @metmirr and it was pointed out that this return here is because the mappings isn't aware of arbitrary new classes which are created in the runtime and must be ignored here.